src/java.naming/share/classes/javax/naming/ldap/StartTlsRequest.java

```diff
@@ -220,23 +220,13 @@ private ConfigurationException wrapException(Exception e) {
      * Acquire the class loader associated with this thread.
      */
     private final ClassLoader getContextClassLoader() {
-        return AccessController.doPrivileged(
-            new PrivilegedAction<ClassLoader>() {
-                public ClassLoader run() {
-                    return Thread.currentThread().getContextClassLoader();
-                }
-            }
-        );
+        PrivilegedAction<ClassLoader> pa = () -> Thread.currentThread().getContextClassLoader();
```

Comment:
We can use here an instance method reference to beautify code a little bit more:
```PrivilegedAction<ClassLoader> pa = Thread.currentThread()::getContextClassLoader;```

Reply:
Good idea, also would fit in with the style of the method just after, `priviligedHasNext()` as that also uses a functional interface.

Reply:
Changes included as described in commit [5d6ecd3](https://github.com/openjdk/jdk/pull/3416/commits/5d6ecd31eb6ed2a63f17b2e798e83b91cb720075)

Reply:
Here again this is not strictly equivalent but AFAIK `Thread::currentThread` doesn't require any permission, so this should be OK.